README.adoc

```diff
@@ -174,6 +174,26 @@ See <<message, Message>>.
 | The label `Perso` does not exist.
 |===
 
+[[severity-level]]
+==== Severity level
+
+The severity can be one of the following:
+
+* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues where the query most probably is wrong and/or should be updated.
+This includes deprecations, experimental features, trying to match on entities with non-existing labels, etc.
+It should be clear what the problem is and it is also good to tell the possible cause, and the consequences, as well as give an example(s) and suggestion(s) for improvement.
+I think the most important part is that "we are (almost) certain it is a problem" and we know how to fix your query. So for:
```

Comment:
```suggestion
The most important part is that "we are (almost) certain it is a problem" and we know how to fix your query. For example:
```

Reply:
Someone was distracted 😄 and forgot to amend it afterward.